api/_lib/parser.ts

```diff
@@ -1,65 +1,43 @@
-import { IncomingMessage } from 'http';
```

Comment:
@saa27 Please use `ESLint` for formatting the code.

You need the following extension (make sure you have the prerequisite mentioned in the document) for the same:
- https://marketplace.visualstudio.com/items?itemName=rvest.vs-code-prettier-eslint (This also has instructions on how to set up your formatter)